app/models/resource_action_workflow.rb

```diff
@@ -32,18 +32,19 @@ def add_request_to_cart
 
   def process_request(state)
     validate_dialog.tap do |result|
-      generate_request(state) if result[:errors].blank?
```

Comment:
@gmcculloug does this look OK? I'm inlining `#generate_request` here to avoid passing the result around. Totally open to an alternative!